0000-template.md

```diff
@@ -21,16 +21,18 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 Why are we doing this? What use cases does it support? What is the expected outcome?
 
-# Guide-level explanation
+#Guide-level-explanation
 [guide-level-explanation]: #guide-level-explanation
 
-Explain the proposal as if it was already included in the language and you were teaching it to another CC+ programmer. That generally means:
+Explain the proposal as if it were already included in the language, or in the kernel, and you were teaching it to another CC+ programmer or Cpcdos OS2.2 contributor. This usually means :
 
 - Introducing new named concepts.
-- Explaining the feature largely in terms of examples.
-- Explaining how CC+ programmers should *think* about the feature, and how it should impact the way they use Rust. It should explain the impact as concretely as possible.
-- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
-- If applicable, describe the differences between teaching this to existing CC+ programmers and new Rust programmers.
+- Explaining the functionality largely through examples.
+- Explaining how CC+ programmers should *think* about the feature and how it should impact on the way they use it. The impact should be explained in as concrete a way as possible.
+- Explain how the feature fits into the kernel and its relationship to different parts of it.
```

Comment:
It should start with explaining (like the previous sentences)
Please use conditionals tense. 
Maybe use and `the relationship between its different parts`